frontend/src/global_styles/layout/work_packages/_table.sass

```diff
@@ -88,12 +88,12 @@
 .work-packages-split-view--tabletimeline-footer
   // Only add padding to lower/right corner
   // to override default margins
-  .pagination
+  .op-pagination
     margin: 0
     padding: 15px 10px 10px 0
 
-  .pagination--pages,
-  .pagination--options
+  .op-pagination--pages,
+  .op-pagination--options
     margin: 0
```

Comment:
Same here, this should just be done inside the `op-pagination` block. If this value differs from the default, then maybe a modifier on  `op-pagination` is warranted. In general, try to avoid selecting elements of other blocks. There are of course always exceptions, but I don't think that's the case here.

Reply:
Also removed